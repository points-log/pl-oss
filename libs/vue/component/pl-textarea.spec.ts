import { shallowMount } from '@vue/test-utils';
import Vue from 'vue';
import Vuetify from 'vuetify';
import PLTextarea from './pl-textarea.vue';

Vue.use(Vuetify);

describe('PLTextarea', () => {
  it('should match the snapshot', () => {
    const wrapper = shallowMount(PLTextarea);
    expect(wrapper.element).toMatchSnapshot();
  });
});
