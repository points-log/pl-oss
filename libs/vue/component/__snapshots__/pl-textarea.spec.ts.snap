// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PLTextarea should match the snapshot 1`] = `
<v-textarea-stub
  backgroundcolor=""
  clearicon="$clear"
  errorcount="1"
  errormessages=""
  hidedetails="auto"
  loaderheight="2"
  messages=""
  outlined="true"
  rowheight="24"
  rows="5"
  rules=""
  successmessages=""
  type="text"
/>
`;
